boot/assets.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/gadget"
 	"github.com/snapcore/snapd/osutil"
+	"github.com/snapcore/snapd/secboot"
```

Comment:
is it an issue that boot now imports secboot given that we use the boot package from the snapd binary for example?

Reply:
AFAIU secboot should be providing a dummy version of the relevant calls.

Reply:
Now it's important to use the `nosecboot` tag to avoid strange errors in systems without snapcore/secboot.